packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -232,6 +229,7 @@ export const DEFAULT_TRANSFORM_REGEX = {
   }
 };
 
+export const VALID_EMAIL_ADDRESS = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Can you document these regex sources? It'd be good to know if this comes from a Cisco doc or a public resource like StackOverflow.

Reply:
i pulled it from web-client and not sure where it's source is from :(